### PR TITLE
Changed to new log system

### PR DESCRIPTION
**Pull request by goodvdh**
Most of it is search and replace of labels with a script. Some by hand.
Obsolete log functions have been removed.

### CODE REVIEW

**Comment by sickpig** [comment]
@goodvdh there tons of formatting warnings that make the travis job fails
e.g. 
https://travis-ci.org/BitcoinUnlimited/BitcoinUnlimited/jobs/332082240#L697

to fix it once manually:
 
```
cd src
cat .formatted-files | xargs -I {} clang-format-3.8  -i -assume-filename=.clang-format {}
```

and to get something automatic append this snippet to the `.git/config`:

```
[filter "bitcoin-clang-format"]
        clean = "contrib/devtools/clang-format.py format-stdout-if-wanted clang-format-3.8 %f"
        smudge = cat
```

this will autoformat your file just after you save it

**Comment by gandrewstone** [comment]
Looks like there may be an issue in the "invalidblockrequest.py" qa test since it failed on 2 runs.  I've restarted both for more datapoints.

EDIT fixed

**Comment by lonoami** [comment]
@sickpig  Thanks for the .git/config snippet.

File: src/main.cpp [hunk 1]
```diff
@@ -6942,21 +6934,21 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
...
+            // LOG(NET, "Reject %s\n", SanitizeString(ss.str()));
             // }
         }
         catch (const std::ios_base::failure &)
         {
             // Avoid feedback loops by preventing reject messages from triggering a new reject message.
-            LogPrint("net", "Unparseable reject message received\n");
-            LogPrint("req", "Unparseable reject message received\n");
+            LOG(NET, "Unparseable reject message received\n");
+            LOG(REQ, "Unparseable reject message received\n");
```
Comment (gandrewstone) [inline src/main.cpp:6943]: combine these into NET | REQ

File: src/main.cpp [hunk 2]
```diff
@@ -2796,11 +2793,11 @@ void static UpdateTip(CBlockIndex *pindexNew)
             pindex = pindex->pprev;
         }
         if (nUpgraded > 0)
-            LogPrintf("%s: %d of last 100 blocks have unexpected version. One example: 0x%x\n", __func__, nUpgraded,
+            LOGA("%s: %d of last 100 blocks have unexpected version. One example: 0x%x\n", __func__, nUpgraded,
                 anUnexpectedVersion);
         if (nUpgraded > 100 / 2)
         {
-            // strMiscWarning is read by GetWarnings(), called by Qt and the JSON-RPC code to warn the user:
+            // strMiscWarning is read by GetWarnings(), called by Qt and the JSON-LOG( code to warn the user:
```
Comment (gandrewstone) [inline src/main.cpp]: accidental replacement in comment

File: src/sync.h [hunk 3]
```diff
@@ -310,8 +310,7 @@ class SCOPED_LOCKABLE CMutexLock
             uint64_t doneTime = GetStopwatch();
             if (doneTime - lockedTime > LOCK_WARN_TIME)
             {
-                LogPrint(
-                    "lck", "Lock %s at %s:%d remained locked for %d ms\n", name, file, line, doneTime - lockedTime);
+                LOG("lck", "Lock %s at %s:%d remained locked for %d ms\n", name, file, line, doneTime - lockedTime);
```
Comment (gandrewstone) [inline src/sync.h]: "lck" -> LCK here

File: src/sync.h [hunk 4]
```diff
@@ -413,8 +412,7 @@ class SCOPED_LOCKABLE CMutexReadLock
             int64_t doneTime = GetStopwatch();
             if (doneTime - lockedTime > LOCK_WARN_TIME)
             {
-                LogPrint(
-                    "lck", "Lock %s at %s:%d remained locked for %d ms\n", name, file, line, doneTime - lockedTime);
+                LOG("lck", "Lock %s at %s:%d remained locked for %d ms\n", name, file, line, doneTime - lockedTime);
```
Comment (gandrewstone) [inline src/sync.h]: "lck" -> LCK

**Pull request merged**